src/Math/Measures.dfy

```diff
@@ -10,10 +10,17 @@ module Measures {
 
   type Probability = x: real | 0.0 <= x <= 1.0
 
-  ghost predicate IsSigmaAlgebra<T(!new)>(eventSpace: iset<iset<T>>, sampleSpace: iset<T>) {
-    && (forall e | e in eventSpace :: e <= sampleSpace)
+  ghost function Complement<T(!new)>(event: iset<T>): iset<T> {
+    iset x: T | x !in event
+  }
+
+  ghost function SampleSpace<T(!new)>(): iset<T> {
+    Complement(iset{})
+  }
+
+  ghost predicate IsSigmaAlgebra<T(!new)>(eventSpace: iset<iset<T>>) {
```

Comment:
I don't think we should remove `sampleSpace` here (and elsewhere). Imo it's confusing for people that read this section for the first time, and it doesn't seem to save us from writing too much code elsewhere.